internal/acctest/provider_factory.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // TestImage is a constant that specifies the default image used in all tests.
-const TestImage = "images:alpine/3.18/amd64"
+const TestImage = "ubuntu-minimal-daily:22.04"
```

Comment:
Should we use `ubuntu-minimal` instead of `ubuntu-minimal-daily` for tests? Or is there any particular benefit of using daily builds?

Reply:
For those tests, it shouldn't matter much as we don't do anything inside the instances themselves, or very little.

I've been leaning toward `-daily` variants lately (in other test settings) because they better represent what users would run. The `-daily` one is essentially the same as the non-daily but with `apt-get update && apt-get upgrade` done inside of it.